_source/handbook/02_drive.md

```diff
@@ -304,6 +304,35 @@ If the form submission is a GET request, you may render the directly rendered re
 
 Servers may also respond to form submissions with a [Turbo Streams](streams) message by sending the header `Content-Type: text/vnd.turbo-stream.html` followed by one or more `<turbo-stream>` elements in the response body. This lets you update multiple parts of the page without navigating.
 
+## Instaclick
+
+Turbo can also speed up network requests by automatically loading links on `mouseenter` or `touchstart` events, and before the user clicks the link. This usually leads to a speed bump of 500-800ms per click navigation.
```

Comment:
Suggestion based on your initial description, might want to take some ideas:

>Turbo can also speed up perceived network requests by automatically prefetching links when users hover over a link before clicking it. This is done on `mouseenter` on desktop and on `touchstart` for mobile devices. This usually leads to a speed bump of 500-800ms per click navigation, since the page will already be there when they click.